src/plugins/intel_cpu/src/nodes/strided_slice.cpp

```diff
@@ -737,6 +758,33 @@ void StridedSlice::StridedSliceCommonExecutor::exec(const std::vector<MemoryCPtr
     });
 }
 
+void StridedSlice::StridedSliceCommonExecutor::execSliceScatter(const std::vector<MemoryCPtr>& srcMemory, const std::vector<MemoryCPtr>& dstMemory) {
+    const uint8_t* srcData = srcMemory[0]->getDataAs<const uint8_t>();
+    const uint8_t* srcUpdates = srcMemory[1]->getDataAs<const uint8_t>();
+    uint8_t* dstData = dstMemory[0]->getDataAs<uint8_t>();
+    cpu_memcpy(dstData, srcData, srcMemory[0]->getSize());
```

Comment:
To consider: There is also `cpu_parallel_memcpy` helper available.
```suggestion
    cpu_parallel_memcpy(dstData, srcData, srcMemory[0]->getSize());
```